### DISCUSSION TITLE
Feature request: floating nodes

### DISCUSSION THREAD

**Discussion by andsav** [xyflow/xyflow#3039] [New Features]
### Please describe the feature that you want to propose

I am looking to add a node at a fixed position on the canvas such that it doesn't move when panning or zooming in/out? Are there any plans to support this in the future? I see the `Node` type has a `positionAbsolute` attribute, but this is not documented.

Thank you!

**Comment by moklick** [comment]
Should it scale when you zoom? Is it connected with other nodes?

**Reply by hoangtrung99** [reply]
@moklick 
Hi sir,
I am having problem with zoom, can I zoom in/out without using `transform: scale` property? Like in figma, it looks like they don't use scale

**Reply by cacoos** [reply]
Figma does not use `transform: scale` because they have elements on a `<canvas />` element.

**Reply by liam-macann** [reply]
I just posted a reference image, which might help make this a little clearer. I think the point is to picture the node as sitting in the `<Panel />` layer, but with edges that interact with the viewport.

**Comment by Salaheldean** [comment]
What then

**Comment by liam-macann** [comment]
I've also come here looking for this! A good reference might be how DaVinci Resolve handles color grading. There is an 'in' and 'out' node that remains fixed to the viewport edge, no matter how you pan or zoom the canvas.

<img width="1247" alt="Screenshot of a color grading flow from DaVinci Resolve" src="https://github.com/xyflow/xyflow/assets/37259090/5c41af2e-d84b-4606-bb50-12bbb8b6cf2d">

I'm not certain this is possible with custom logic at present, short of just negating the transform. I'd be super interested to hear if anyone has any approaches, even if a dedicated feature won't make it to core.
